src/app/components/token-network/token-network.component.ts

```diff
@@ -307,9 +310,11 @@ export class TokenNetworkComponent implements OnInit, OnDestroy {
 
     mintToken(token: UserToken, component: TokenNetworkActionsComponent) {
         component.requestingTokens = true;
+        const amount =
+            token.decimals >= 2 ? amountFromDecimal(0.01, token.decimals) : 1;
```

Comment:
This means that if the token has no decimals you will get 1 token right? In that case I cannot send less than 1 token and each mint gives me 1 token. 

Maybe it should be 0.01 if you have 18 decimals and maybe as you go up to 0 decimals it should increase to something like 100 or 1000 tokens. What do you think about it?

Reply:
I didn't think of that case. It makes sense to scale it like that, so I will implement it with up to 1000 tokens.